src/app/tests/suites/TestBasicInformation.yaml

```diff
@@ -79,7 +79,6 @@ tests:
                   19,
                   0xFFF8, # GeneratedCommandList
                   0xFFF9, # AcceptedCommandList
-                  0xFFFA, # EventList
```

Comment:
This will now fail if the build flag is turned on, right?  Probably OK, but not ideal....

Reply:
Yes, it will fail when the flag is turned on. I made this quick patch for the SVE release today. I'll make this implementation cleaner next week.